tools/platforms/AndroidPlatform.hx

```diff
@@ -364,23 +353,14 @@ class AndroidPlatform extends PlatformTarget
 
 	public override function rebuild():Void
 	{
-		var armv5 = (/*command == "rebuild" ||*/
-			ArrayTools.containsValue(project.architectures, Architecture.ARMV5)
-			|| ArrayTools.containsValue(project.architectures, Architecture.ARMV6));
-		var armv7 = (command == "rebuild" || ArrayTools.containsValue(project.architectures, Architecture.ARMV7));
-		var arm64 = (command == "rebuild" || ArrayTools.containsValue(project.architectures, Architecture.ARM64));
-		var x86 = (ArrayTools.containsValue(project.architectures, Architecture.X86));
-		var x64 = (command == "rebuild" || ArrayTools.containsValue(project.architectures, Architecture.X64));
-		
 		var commands = [];
 		var minSDKVer = 21;
 		var platformDefine = '-DPLATFORM_NUMBER=$minSDKVer';
-		
-		if (armv5) commands.push(["-Dandroid", platformDefine]);
-		if (armv7) commands.push(["-Dandroid", "-DHXCPP_ARMV7", platformDefine]);
-		if (arm64) commands.push(["-Dandroid", "-DHXCPP_ARM64", platformDefine]);
-		if (x86) commands.push(["-Dandroid", "-DHXCPP_X86", platformDefine]);
-		if (x64) commands.push(["-Dandroid", "-DHXCPP_X86_64", platformDefine]);
+
+		commands.push(["-Dandroid", "-DHXCPP_ARM64", platformDefine]);
+		commands.push(["-Dandroid", "-DHXCPP_ARMV7", platformDefine]);
+		commands.push(["-Dandroid", "-DHXCPP_X86_64", platformDefine]);
+		commands.push(["-Dandroid", "-DHXCPP_X86", platformDefine]);
```

Comment:
It would be useful to do the same for the rebuild as well:
- don't rebuild armv7 and x86_32 by default
- rebuild if the project contains them, like we used to do for x86_32